life-and-letters/modernized/11-chapter-10.adoc

```diff
@@ -28,16 +28,16 @@ in their language and style, that portions of them were of his composition.
 From the records of the yearly meeting in 1753,
 it appears that he then proposed the establishment of a
 yearly meeting of ministers and elders,
-and several minutes occur on the subject The proposal made by our friend Samuel
+and several minutes occur on the subject&hellip; The proposal made by our friend Samuel
 Fothergill respecting the establishment of a yearly meeting of ministers and elders,
 being reconsidered,
 was referred to the consideration of the committee for drawing up the epistle John Fry,
 from the committee appointed to consider the proposal made by
 Samuel Fothergill with respect to the holding a general
 meeting for ministers and elders annually in this city,
-brought in a report, in which they say, That having solidly considered the proposition,
+brought in a report, in which they say, "`That having solidly considered the proposition,
```

Comment:
I wasn't sure if this should have quotation marks because it doesn't seem to be an actual direct quote. But the first letter was capitalized, and in GF's stuff, he (or someone else) used quotes like this all the time even though they weren't truly direct quotes.